docs: kwaliteitsaanpak (#2840)
nl-design-system/documentatie: docs/project/kwaliteitsaanpak/veilig-werken.mdx

---
title: Veilig werken
hide_title: true
hide_table_of_contents: false
sidebar_label: Veilig werken
sidebar_position: 4
pagination_label: Veilig werken
description: Veilig werken is belangrijk voor NL Design System, dit is wat je kunt doen.
---

import { Paragraph } from "@utrecht/component-library-react/dist/css-module";

{/* @license CC0-1.0 */}

# Veilig werken

<Paragraph lead>
  Samen houden we NL Design System veilig. Het is belangrijk dat je weet hoe je veilig kunt werken met je eigen devices
  en accounts, en dat je geen onverantwoorde risico's neemt.
</Paragraph>

## Bescherm je device

Zorg dat anderen geen gebruik kunnen maken van jouw device, en alle rechten die daarbij horen.

Lock je devices wanneer je ze niet gebruikt. Het best is om je device automatisch snel te locken bij inactiveit.

Kies een moeilijk wachtwoord voor je device, en kies een pincode van tenminste 6 cijfers voor je smartphone.

> Gebruik waar mogelijk gebruik van biometrische identificatie, zoals een vingerafdruk of gezichtsherkenning.

Reviewer: ```suggestion
Gebruik waar mogelijk biometrische identificatie, zoals een vingerafdruk of gezichtsherkenning.
```

## Encryptie

Gebruik encryptie op je device. Op macOS kun je FileVault gebruiken. Op Windows is dat BitLocker. Lees op het internet hoe je dit instelt:

- [Apple: Protect data on your Mac with FileVault
  ](https://support.apple.com/en-gb/guide/mac-help/-mh11785/mac)
- [Microsoft: Device Encryption in Windows](https://support.microsoft.com/en-us/windows/device-encryption-in-windows-cf7e2b6f-3e70-4882-9532-18633605b7df)

## Unieke en veilige wachtwoorden

Voorkom dat uitgelekte wachtwoorden ook werken bij andere accounts. Voorkom dat makkelijke wachtwoorden gekraakt kunnen worden.

Gebruik unieke wachtwoorden voor elke account, en gebruik wachtwoorden die zo complex en lang genoeg zijn om veilig te zijn. Moet je tijdens je werk regelmatig een wachtwoord intypen? Bijvoorbeeld bij het pushen naar GitHub? Misschien is je wachtwoord dan niet veilig genoeg.

> Met een password manager is het mogelijk om unieke wachtwoorden te gebruiken voor elke account, die bovendien erg complex zijn.

Reviewer: Misschien is het slim dit in een sectie te doen?
```suggestion

### Gebruik een wachtwoord manager
Met een wachtwoord manager is het mogelijk om unieke wachtwoorden te gebruiken voor elke account, die bovendien erg complex zijn, zonder dat je deze hoeft te onthouden.

Gebruik je nog geen wachtwoord manager? Overweeg dan toch er een te gebruiken. Het is de meest praktische manier om aan alle eisen te voldoen. 
```

Als je geen password manager gebruikt, dan is dit waarschijnlijk wel nodig: het is de meest praktische manier om aan alle eisen te voldoen.

## Deel geen persoonlijke accounts

Deel geen inloggegevens van persoonlijke accounts met collega's. Geef anderen geen controle over je scherm bij remote samenwerken.

## Gebruik een tijdslimiet

Zorg dat je alleen toegang heb zo lang als het nodig is.

Wanneer je inlogt met een account met extra veel rechten, dan kun je dit doen in een incognito-browser. Dan ben in elk geval weer uitgelogd wanneer je de tab of browser sluit.

> Wanneer je access tokens instelt, zorg dan dat de toegang na een tijdje weer verloopt. Bijvoorbeeld volgende dag al. Of eind van het kwartaal. Maak nooit tokens aan die toegang hebben tot voorbij een einddatum van je contract. Gebruik geen access tokens die nooit verlopen, wanneer er een alternatief is.

Reviewer: ```suggestion
Wanneer je access tokens instelt, zorg dan dat de toegang na een tijdje automatisch weer verloopt. Gebruik geen access tokens die nooit verlopen wanneer er een alternatief is. Kies dan bijvoorbeeld de volgende dag als je alleen een test doet, elk kwartaal om extra veilig te zijn, of eind van je contract of de opdracht.
```

> Wanneer je access tokens instelt, zorg dan dat de toegang na een tijdje weer verloopt. Bijvoorbeeld volgende dag al. Of eind van het kwartaal. Maak nooit tokens aan die toegang hebben tot voorbij een einddatum van je contract. Gebruik geen access tokens die nooit verlopen, wanneer er een alternatief is.

Reviewer: Zou dit eigenlijk gewoon een losse sectie moeten zijn zodat je die extra makkelijk tegenkomt? Dan zou je daar zelfs nog een sectie bij kunnen zetten die gaat over environment secrets etc.?

Author: Dit kan denk ik het best later in het handboek onder "Infrastructuur"

https://github.com/nl-design-system/documentatie/issues/4204
